main.go

```diff
@@ -73,6 +99,28 @@ func main() {
 				"DO NOT USE in production",
 		)
 		vault, err = deterministic.NewStore(numDeterministicKeys)
+	case "mnemonic":
+		log.Warn(
+			"Using a mnemonic to recover keys from",
+		)
+
+		if mnemonicFile == "" {
+			log.Fatal("You must provide a --mnemonic-file")
+		}
+
+		content, err := ioutil.ReadFile(mnemonicFile)
+		if err != nil {
+			logrus.Info("File reading error", err)
+			return
+		}
+		mnemonicPhrase := string(content)
+		logrus.Info("Contents of file: ", mnemonicPhrase)
+
+		vault, err = mnemonic.NewStore(
```

Comment:
Unhandled error `err`.

Reply:
I am not handling it here because its handled a bit later [here](https://github.com/prysmaticlabs/remote-signer/blob/d5c6b66fb36221ca43a9a6425abb87680faac957/main.go#L127). So instead of failing for each type of keystore, there is a common fail for all of them. Was trying to follow the [deterministic case](https://github.com/prysmaticlabs/remote-signer/blob/master/main.go#L75) where its neither handled. What do you think?

Reply:
Let's leave it like this. Thanks for explaining.